okapi-core/src/main/raml/okapi.raml

```diff
@@ -1128,6 +1128,31 @@ types:
         body:
           text/plain:
 
+/_/proxy/import/modules:
+  description: Import modules
```

Comment:
The related endpoints should have related descriptions:

```
/_/proxy/modules:
  description: Proxy modules service
```
```
/_/proxy/import/modules:
  description: Proxy modules service, list import
```

Reply:
ok